test/operators/update-fee.ts

```diff
@@ -0,0 +1,183 @@
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod);
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[2]).executeOperatorFee(1
+    )).to.emit(ssvNetworkContract, 'OperatorFeeExecution');
+  });
+
+  it('Execute declared fee gas limits', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod);
+    await trackGas(ssvNetworkContract.executeOperatorFee(1), [GasGroup.REGISTER_OPERATOR]);
+  });
+
+  it('Execute declared fee of operator I do not own reverts "CallerNotOwner"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await expect(ssvNetworkContract.connect(helpers.DB.owners[1]).executeOperatorFee(1
+    )).to.be.revertedWith('CallerNotOwner');
+  });
+
+  it('Execute declared fee without a pending request reverts "NoPendingFeeChangeRequest"', async () => {
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('NoPendingFeeChangeRequest');
+  });
+
+  it('Execute declared fee too early reverts "ApprovalNotWithinTimeframe"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod - 10);
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('ApprovalNotWithinTimeframe');
+  });
+
+  it('Execute declared fee too late reverts "ApprovalNotWithinTimeframe"', async () => {
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, initialFee + initialFee / 10), [GasGroup.REGISTER_OPERATOR]);
+    await progressTime(helpers.CONFIG.declareOperatorFeePeriod + helpers.CONFIG.executeOperatorFeePeriod + 1);
+    await expect(ssvNetworkContract.executeOperatorFee(1
+    )).to.be.revertedWith('ApprovalNotWithinTimeframe');
+  });
+
+  //Dao
+  it('DAO get fee increase limit', async () => {
+    expect(await ssvNetworkContract.getOperatorFeeIncreaseLimit()).to.equal(helpers.CONFIG.operatorMaxFeeIncrease);
+  });
+
+  it('DAO get declared fee', async () => {
+    const newFee = initialFee + initialFee / 10;
+    await trackGas(ssvNetworkContract.declareOperatorFee(1, newFee), [GasGroup.REGISTER_OPERATOR]);
+    const [feeDeclaredInContract] = await ssvNetworkContract.getOperatorDeclaredFee(1);
+    expect(feeDeclaredInContract).to.equal(newFee);
+  });
+
+  it('DAO get declared fee period', async () => {
+    expect(await ssvNetworkContract.getDeclaredOperatorFeePeriod()).to.equal(helpers.CONFIG.declareOperatorFeePeriod);
+  });
+
+  it('DAO get execute fee period', async () => {
+    expect(await ssvNetworkContract.getExecuteOperatorFeePeriod()).to.equal(helpers.CONFIG.executeOperatorFeePeriod);
+  });
+  
+  it('DAO increase the fee limit emits "OperatorFeeIncreaseLimitUpdate"', async () => {
```

Comment:
Increase what fee?
what if they decrease the fee?
we have something called operatorFeeIncreaseLimit thats default set to 10%
I can change this number from 10% to 8% it would be a decrease
Please update title